main.js

```diff
@@ -0,0 +1,31 @@
+const keys = document.querySelectorAll(".key");
+
+window.addEventListener("keydown", playSound);
+keys.forEach((key) => key.addEventListener("transitionend", removeTransition));
+keys.forEach((key) => key.addEventListener("click", clickMouse));
+
+function playSound(e) {
+  const key = document.querySelector(`.key[data-key="${e.keyCode}"]`);
+  const audio = document.querySelector(`audio[data-key="${e.keyCode}"]`);
+  if (!audio) return;
```

Comment:
It's worth using positive condition
`if (audio) {
   playAudioAddClass(audio, key);
}`